Core/HLE/sceNet.cpp

```diff
@@ -313,6 +352,13 @@ void __NetDoState(PointerWrap &p) {
 		apctlStateEvent = -1;
 	}
 	CoreTiming::RestoreRegisterEvent(apctlStateEvent, "__ApctlState", __ApctlState);
+	if (s >= 6) {
+		Do(p, apctlPollEvent);
+	}
+	else {
+		apctlPollEvent = -1;
+	}
+	CoreTiming::RestoreRegisterEvent(apctlPollEvent, "__ApctlPollEvent", __ApctlPollEvent);
```

Comment:
Currently, i'm having crash issue due to this newly added `RestoreRegisterEvent` when loading old SaveState on Def Jam Fight for NY
![image](https://user-images.githubusercontent.com/7974720/116355185-b3469080-a823-11eb-8106-1d09278f8a9f.png)
![image](https://user-images.githubusercontent.com/7974720/116355355-ed179700-a823-11eb-88cd-a9ba386f6bae.png)
Which causes an issue during `__UsbMicDoState`

May be we should create newly registered events after savestate fully loaded? 
So it won't insert newly registered event id in the middle and messed up id(s) of existing event(s) that might be active when the savestate were created, thus causing the id pointing to the wrong event function.

Reply:
It should allocate a new one at the end - the problem is that it does need to set the variable to the ID.

The problem here is that you're calling RestoreRegisterEvent when you never called RegisterEvent in the first place.  That's not valid and ends up causing this error.

-[Unknown]

Reply:
I copy that `RestoreRegisterEvent` from other DoState, the event itself will be created when Apctl being initialized, but this game doesn't even use Apctl (usually used for infrastructure) because it doesn't support infrastructure, thus never gets registered.

Reply:
It has to be initialized in init now and then everything works.

-[Unknown]

Reply:
The `__NetDoState` didn't have any issue, but because there is a new register event in `__NetDoState`, the following DoState have a possibility to crashed (in this case the `__UsbMicDoState` getting the error), probably because the id got shifted